IFC4x3/Properties/r/RoadOwner_3$AHwHvzf1kP7QyDaGuXOs/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RoadOwner_3AHwHvzf1kP7QyDaGuXOs" Name="RoadOwner" UniqueId="ff291e91-e7da-41b9-91da-f0d910e21636" PropertyType="p_enumeratedvalue" PrimaryDataType="IfcLabel">
```

Comment:
1. why it is a PEnum??
2. Use `IfcRelAssignsToActor` with relevant `IfcActor` and `IfcActorRole` e.g. OWNER or ROADOWNER

Reply:
Removed https://github.com/bSI-InfraRoom/IFC-Specification/pull/164/commits/ab85f083c2717d30ef42c6a47dcce352a5b16e89